dellmont-credit-checker.sh

```diff
@@ -202,7 +202,9 @@ check_credit_level() {
 				local PREVCREDIT=("2000-01-01 00:00 0")
 			fi
 			echo -e "`date +"%Y-%m-%d %T"`\t$CREDITCENTS">>"$6"
-			local CREDITFALL=$((${PREVCREDIT[2]}-$CREDITCENTS))
+			# remove leading spaces if any, and add 10# so to make it work with credit like: "093" " 201"
+			temp_var=`echo $CREDITCENTS | sed -e 's/^[ \t]*//'`
+                        local CREDITFALL=$((10#${PREVCREDIT[2]}-10#$temp_var))
```

Comment:
yes, of course.  I used a temp var just because I did not have time to check if that var was used some place else (just in case; I was in a hurry ;)